system-int-test/test/cmr/system_int_test/search/generics_test.clj

```diff
@@ -358,3 +360,91 @@
 
                    "explicit UMM JSON version through suffix"
                    (format "application/vnd.nasa.cmr.umm+json;version=%s" (generics/current-generic-version (keyword concept-type-string))))))))))
+
+;; Test that generic draft collections non json records can be ingested and searched.
+(deftest all-generic-non-json-collection-draft-search-results-test
+  (let [dif10-file {mt/dif10 (slurp (io/resource "CMR-5138-DIF10-DataDates-Provided.xml"))}
+        echo10-file {mt/echo10 (slurp (io/resource "echo10-samples/collection.echo10"))}
+        iso-file {mt/iso19115 (slurp (io/resource "iso-samples/5216_IsoMends_Collection.xml"))}
+        files (concat dif10-file echo10-file iso-file)]
+    (doseq [file files]
+      (let [mime-type (first file)
+            metadata (second file)
+            sanitized-collection (cmr.umm-spec.umm-spec-core/parse-metadata {:ignore-kms-keywords true} :collection mime-type metadata)
+            name (:ShortName sanitized-collection)
+            native-id (format "Generic-Test-%s" (UUID/randomUUID))
+            concept-type-string "collection-draft"
+            post-results (gen-util/generic-request nil "PROV1" native-id concept-type-string metadata :put mime-type)
+            body (json/parse-string (:body post-results) true)
+            concept-id (:concept-id body)
+            revision-id (:revision-id body)
+            plural-concept-type-name (inf/plural concept-type-string)]
+        (index/wait-until-indexed)
+
+        (testing "Check that test the document ingested before going forward with tests"
+          (is (= 201 (:status post-results)) "failed to ingest test record"))
+
+        (testing "Testing generics name parameter search"
```

Comment:
Is this more fitting; as I understand drafts are generics but, this is really draft collections
```suggestion
        (testing "Testing draft collections name parameter search"
```

Reply:
Yes, while the record that is being searched is a collection-draft, the search parameter is still a generic search parameter search.  I have changed the comment.